server/src/main/java/com/objectcomputing/checkins/services/checkins/CheckInServicesImpl.java

```diff
@@ -18,62 +20,80 @@
 
     @Inject
     private MemberProfileRepository memberRepo;
```

Comment:
Private field injection is also bad. This can also be resolved by using constructor injection.

Reply:
I should add that the class must be annotated with @Singleton to use constructor injection

Reply:
Yeah, I would change all your dependency fields to private and inject them via constructor.